proxy/envoy/discovery.go

```diff
@@ -151,6 +151,12 @@ type hosts struct {
 type host struct {
 	Address string `json:"ip_address"`
 	Port    int    `json:"port"`
+	Tags    tags   `json:"tags,omitempty"`
```

Comment:
I see now. Thats because an empty pointer is nil but an empty tag is tag{}.

Probably makes sense to go with the pointer.